src/Integration.Vsix/SonarLintTagger/IssueConsumer.cs

```diff
@@ -92,12 +88,10 @@ public void Accept(string path, IEnumerable<IAnalysisIssue> issues)
 
             Debug.Assert(issues.All(IsIssueFileLevelOrInAnalysisSnapshot), "Not all reported issues could be mapped to the analysis snapshot");
 
-            var newIssues = issues
+            allIssues = issues
```

Comment:
Previously, all of the different 'analyzers' were publishing to the same issue consumer, so it needed to be accumulating. Now, we only have one `analyzer`, which posts all issues for all languages for the file, meaning we no longer need to accumulate